commands/initial/templates/package.json

```diff
@@ -15,7 +15,9 @@
     "prebuild": "rimraf dist out-tsc",
     "start": "webpack-dev-server --open --config ./webpack/webpack.dev.js",
     "tagVersion": "np --no-publish",
-    "test": "node ./tasks/test"
+    "test": "node ./tasks/test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
```

Comment:
@nvdnkpr thanks for your continued participation in this project!

wouldn't being able to run `npm test e2e` satisfy the same thing as `npm run e2e`? We can update the test script to understand the arguments passed to it/add a configuration file so that the Selenium server port can be customized.

Does that satisfy your concerns?